File: Back/NodeJS/src/domain/bets/events/BetCreated.ts
import { IDomainEvent } from "../../IDomainEvent";
import { MemberId } from "../../members/MemberId";
import { BetId } from "../BetId";

export class BetCreated implements IDomainEvent {
    constructor(public BetId: BetId, public MemberId: MemberId){}
    Type: string = BetCreated.name;
}

File: Back/.Net/Modules/Bets/BetFriends.Bets.Domain/Events/BetCreated.cs
using BetFriends.Bets.Domain.Bets;
using BetFriends.Bets.Domain.Members;
using BetFriends.Shared.Domain;

namespace BetFriends.Bets.Domain.Events;

public record BetCreated(BetId BetId, MemberId OwnerId, int Coins) : IDomainEvent;
